ztp/gitops-subscriptions/argocd/example/policygentemplates/common-ranGen.yaml

```diff
@@ -18,50 +18,50 @@ spec:
     - fileName: validatorCRs/informDuValidator.yaml
       complianceType: musthave
       remediationAction: inform
-      policyName: "du-validator-policy"
+      policyName: "config-policy"
```

Comment:
@Missxiaoguo is there any reason this policy needs to be separate from the config policies or can it be combined with them?

Reply:
We shouldn't combine this policy with other config policies as this is an inform policy but others are enforce and policyGen doesn't support have different types of remedicationAction in one policy.

Even we will eventually have the default to inform, we should leave it separated for LO to know this policy shouldn't be copied for enforce.

Reply:
+1